cmd/start.go

```diff
@@ -64,11 +64,12 @@ func start(newProxy ProxyFactory) error {
 		},
 	}
 
-	if inClusterK8sCfg, _ := rest.InClusterConfig(); inClusterK8sCfg != nil {
+	if inClusterK8sCfg, err := rest.InClusterConfig(); inClusterK8sCfg != nil {
```

Comment:
Not related this feature but this fixes an issue that log in the else-if block in [line 76](https://github.com/Twingate/kubernetes-access-gateway/pull/24/files#diff-e3ad76770e5ac8ed654e8c5585782469e67e0d981adb9153604109263f22e93eR76) is always printed when the gateway is running outside the k8s cluster.